docs/userGuide/syntax/headers.mbdf

```diff
@@ -23,6 +23,18 @@ In the page that you want to include the header:
 ```
 </div>
 
+You can also choose to add a <trigger for="pop:progress-indicator">reading progress indicator</trigger> to let the reader know how far along they are in the page.
```

Comment:
Due to how our current search works, your feature won't be discoverable if the authors attempt a search on our documentation website. I suggest giving a heading to your section:

`### Adding progress indicators`

Reply:
Sure, will do 👍